core/login.go

```diff
@@ -98,26 +104,27 @@ func createAccount(username string, ip string) bool {
 			fmt.Println(err.Error())
 			continue
 		}
-		if password == string(bytePassword) {
-			return dbAddUser(username, password, ip)
+		if password == string(bytePassword) && dbAddUser(username, password, ip) {
```

Comment:
While cool, im not a fan of this. If dbAddUser fails it will print "Passwords don't match"

Reply:
Good catch. I'll move it back down.